AVI-SHUKLA1110.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/88367072/219391424-f79f32ac-0b3f-42ad-a19c-3fb0cfacb186.png",
```

Comment:
I recommend more of an image like this. What do you think? 🙂

![image](https://user-images.githubusercontent.com/51391473/219433518-19a7d654-552c-4cee-a2a0-3153108ed35e.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/219433518-19a7d654-552c-4cee-a2a0-3153108ed35e.png",
```